doc/main/tbb_userguide/Mac_OS.rst

```diff
@@ -15,18 +15,34 @@ The following table describes the subdirectory structure for macOS\*, relative t
         - Location     
         - Environment Variable     
       * - Header files     
-        - | ``include/oneapi/tbb.h``
- 	  | ``include/oneapi/tbb/*.h``     
+        - | ``<tbb_install_dir>/include/oneapi/tbb.h``
+ 	  | ``<tbb_install_dir>/include/oneapi/tbb/*.h``     
         - ``CPATH`` 
       * - Shared libraries
-        - ``lib/<lib><variant>.<version>.dylib``
+        - ``<tbb_install_dir>/lib/<lib><variant>.<version>.dylib``
         - | ``LIBRARY_PATH``
 	  | ``DYLD_LIBRARY_PATH``
+      * - Symbolic links
+        - | ``<tbb_install_dir>/lib/<lib><variant>.dylib`` -> ``<tbb_install_dir>lib/<lib><variant>.<ver_major>.dylib``
```

Comment:
what is  -> for?

Reply:
"->" means that the left part is a link to the right part.